scripts/project-setup-scripts/create-custom-iam-roles-for-projects.sh

```diff
@@ -17,6 +17,6 @@ then
        "
   exit 1
 else
-  gcloud iam roles create terra-workspace-can-compute --organization=${ORGANIZATION_ID} --file=terra-workspace-can-compute.yaml
-  gcloud iam roles create terra-billing-project-owner --organization=${ORGANIZATION_ID} --file=terra-billing-project-owner.yaml
+  gcloud iam roles create terra_workspace_can_compute --organization=${ORGANIZATION_ID} --file=terra-workspace-can-compute.yaml
+  gcloud iam roles create terra_billing_project_owner --organization=${ORGANIZATION_ID} --file=terra-billing-project-owner.yaml
```

Comment:
google doesn't allow hyphens in role IDs

Reply:
How did the script work before? Did it? Or was there a breaking google change?

Reply:
I had tested the commands but toward the end of the review process, the name of the role name was changed and that was never tested. Since the PR was merged when I was on vacation, I didn't run the scripts.